docs/reference.md

```diff
@@ -759,6 +759,11 @@ Content can
 be [MessageML](https://docs.developers.symphony.com/building-bots-on-symphony/messages/overview-of-messageml) with
 the `<messageML>` tags or can be simple text too (<messageML> are automatically added if needed).
 
+Content can either set directly in the swadl file as plain text or it can be references from an external file. Both freemarker or mml.xml
```

Comment:
```suggestion
Content can either set directly in the SWADL file as plain text or it can be references from an external file. Both freemarker or mml.xml
```